src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,52 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  let currentTab = tabs.find(tab => tab.id === selectedTabId);
```

Comment:
```suggestion
  let currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```
instead of code below